File: Easy/2026-07-07-3754-Concatenate-Non-Zero-Digits-and-Multiply-by-Sum-I/solution.py
class Solution:
    def concatenateNonZeroDigitsAndMultiply(self, n: int) -> int:
        digits = [int(d) for d in str(n) if d != '0']
        if not digits:
            return 0
        x = int(''.join(map(str, digits)))
        sum_digits = sum(digits)
        return x * sum_digits

File: Easy/2026-07-07-3754-Concatenate-Non-Zero-Digits-and-Multiply-by-Sum-I/solution.js
var concatenatedSum = function(n) {
    const digits = n.toString().split('').filter(d => d !== '0');
    if (digits.length === 0) return 0;
    
    const x = parseInt(digits.join(''), 10);
    const sum = digits.reduce((acc, d) => acc + parseInt(d, 10), 0);
    
    return x * sum;
};